contents/blog/startup-finance-without-finance.md

```diff
@@ -0,0 +1,236 @@
+
+Once you're done, this is how to get it into Pry: 
+
+- Go to Models -> Expense
+- Next to 'Inputs', hit '+' and select the 'Import' option
+- Select the same Google Sheet again
+- Import each 'Range' you created in your Google Sheet
+- These should now show up in your model Input on the left hand side
+- Create a new formula for each range, and set each value to just be equal to the input you've imported (ie. create a formula called 'IT Expense' = 'IT Expense import')
+- In the right hand pane, create a new Output for each line item – map each formula output to the relevant item in your budget
+
+Pheww! Again, make sure you refresh the inputs on this Model page if you make changes to your Google Sheet in the future.
+
+If this has worked, you should see all the expenses being projected forward in your main Financials tab in Pry as well.
+
+> A note on scenarios: these are a really important part of our financial planning process, and are easy to build in Pry. In particular, we regularly check our '[default alive](http://www.paulgraham.com/aord.html)' scenario to ensure we're on track. I'll cover this topic in a future post. 
+
+#### Hiring plan
+
+Finally, head over to Pry's 'Hiring Plan' tab. This section is pretty straightforward – fill in your current team, plus projected future hires. In theory you could do this in Sheets as well and import again, but I like the fine-grained control in Pry. Plus under the Settings cog, you can do things like add a % for additional tax etc. so you get the fully loaded team member cost, and model future pay rises in. 
+
+You should now have a nice financial plan set up in Pry. I'm not going to cover everything that Pry does in this guide, so I'd recommend you check out [their docs](https://pry.co/docs/introduction) instead. 
+
+### Budget vs. actuals
+
+Once a month, after we've received our monthly financial report from Fondo (remember that?), we review how we performed against our budget in Pry. You can also reallocate revenue and expenses to different categories in Pry by simply dragging and dropping, but be aware that this doesn't change anything in QuickBooks. 
+
+Our process is pleasingly simple:
+
+1. Open up your main Financials view in Pry
+2. On the left, select the Planned vs. Actuals tab
+3. Next to the name of the month you want to look at, hover over and a little arrow will appear saying 'Set Budget Start Date' – click on it
+4. This will then bring up a two column view of the month you are looking at
+5. We look at how we performed against our cashflow forecast (seriously, it's basically the only thing we care about)
+6. We then scroll through our plan and pay particular attention to anything that looks like it's gone unusually over or under budget – if anything is in the wrong category, you can just drag and drop it into the correct one
```

Comment:
```suggestion
Our process is pleasingly simple:

1. Open up your main Financials view in Pry

2. On the left, select the Planned vs. Actuals tab

3. Next to the name of the month you want to look at, hover over and a little arrow will appear saying 'Set Budget Start Date' – click on it

4. This will then bring up a two column view of the month you are looking at

5. We look at how we performed against our cashflow forecast (seriously, it's basically the only thing we care about)

6. We then scroll through our plan and pay particular attention to anything that looks like it's gone unusually over or under budget – if anything is in the wrong category, you can just drag and drop it into the correct one
```